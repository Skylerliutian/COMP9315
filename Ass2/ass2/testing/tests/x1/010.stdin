N A 50
S A
u A 20
u A 22
u A 18
O A

i A 17
i A 18
i A 19
i A 20
i A 21
i A 22
i A 23