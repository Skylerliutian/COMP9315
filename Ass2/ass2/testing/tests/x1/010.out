11111111111111111111111111111111101010111111111111111111
True
False
True
False
True
False
True
